infra/test/cardano-sl-infra-test.cabal

```diff
@@ -0,0 +1,55 @@
+name:                cardano-sl-infra-test
+version:             1.1.0
+synopsis:            Cardano SL - generators for cardano-sl-infra
+description:         This package contains generators for the infrastructural data types used in Cardano SL.
+license:             MIT
+license-file:        LICENSE
+author:              Serokell
+maintainer:          hi@serokell.io
+copyright:           2018 IOHK
+category:            Currency
+build-type:          Simple
+cabal-version:       >=1.10
+
+library
+  exposed-modules:
+                       Test.Pos.Infra.Gen
+
+  build-depends:       base
+                     , bytestring
+                     , cardano-sl-core
+                     , cardano-sl-infra
+                     , hedgehog
+                     , kademlia
+                     , memory
+                     , time-units
+                     , universum
+
+  default-language:    Haskell2010
+
+  default-extensions:  DeriveDataTypeable
```

Comment:
I need to take out the `default-extensions` as well and stick to specifying them at the module-level.